### ISSUE TITLE
Article: Inherents vs Digest Data – where should Authorship data go?

### ISSUE THREAD

**Issue by gnunicorn** [substrate-developer-hub/knowledgebase#3]
People are wondering which of the two should be used for what information. In particularly about where Authorship data (of a block) should/needs to go. An article outlining the difference between the two types of data and what considerations to take into account when making that decision would be great.

See: https://radicle.community/t/block-rewards-in-the-registry/152/6

**Comment by JoshOrndorff** [comment]
Indeed, I'm still wondering what the difference between inherents and preruntime digests are. When should I use one over the other. Is there any example of something that can use one but not the other?

**Comment by joepetrowski** [comment]
Removing docs label because _everything_ in this repo is docs.

**Comment by joepetrowski** [comment]
![image](https://user-images.githubusercontent.com/25483142/80683033-d3822900-8ac3-11ea-960e-4a2adcc389ee.png)
